### DISCUSSION TITLE
BBS02 30A broken after a few km

### DISCUSSION THREAD

**Discussion by aaron-manning** [danielnilsson9/bbs-fw#148] [General]
A warning for anyone wanting to try the same

### Specs
BBS02B (purchased 1 year ago) 750W 48v model.
36v 15AH 30A battery.
Motor and battery have less than 50km of total usage.

### Config
30v min voltage to allow 36v nominal battery
Max amps set to 30
PAS disabled
Throttle settings default (1% start current)

### Why 30A?
I had read online that any BBS02B produced within the last several years will have upgraded MOSFETs capable of more amps and more heat (although my motor/controller has never gotten hot, I keep an eye on it). I have also read that older BBS02 with fewer and poorer quality MOSFETs were coming shunted from the factory and produced 28-30A. If the old model with few and crappy MOSFETs handled 30A, the new model with improved hardware should, right?

I looked up what the community typically consider as a safe max voltage/amp combo, which seemed to be to not go over 22A with a 52v battery. I ran motor simulations and determined that the phase amps and total power of a 36v battery at 30A is equal to a 52v battery limited to 22A at the controller. The 36v battery would peak in power (around 1000W) at a crank rpm of around 65. The 52v config would be identical up until this point, but would continue to produce power and reach a higher rpm (and thus higher total power)

I had thought that as 36v 30A should produce less total power than 52v 22A, there should be no problem. Obviously I was wrong.

### What happened
I was riding in the woods on rough terrain, but mostly flat. Ambient temperature was about 25c, motor case temperature about 40c after 15 mins riding. I came to an (almost) stop on a gravel road in the lowest gear, hit the throttle hard (new toy, only recently upped the amps and wanted to see how much torque) and the power cut off.

No grinding or clicking sounds, just shut off and has shown no signs of life since.

### I have tried
1. Checking the battery. It appears fine. I tried plugging a 30v input to the motor, which should have been enough voltage to light up the programming cable, but no signs of life.
2. Bypassing the display. Still no power.
3. Checking for continuity on the battery leads. There is no short there.
4. Rolling the bike backwards and checking for resistance. It seems fine and I can hear the motor turning normally. This suggests it might not be blown MOSFETs.

### Next steps
1. Remove the controller and continuity check the phases. Check that power can reach the controller.
2. Check the motor core for blown windings. People say there should be a smell when these burn, but I didn't notice anything.

Does anyone have any tips to help me diagnose this? I want to avoid pulling the motor core out if possible.

**Comment by Gonespa** [comment]
Hi,
I have my 750w 48v unit configured at 33A and other than a few cutouts for about a couple of seconds i had no problem. And I beat it harder than you , I go up ramps at snail pace for several seconds at full throttle. I also run it at locked rotor full throttle several times for some experiments I did.
I doubt you have a melted winding issue.

Did you connect your unit to the config util to see if any error message shows in the log?

**Reply by aaron-manning** [reply]
There's no power at all. When I plug the programming cable into the bike and connect the battery (or another power source), the programming cable doesn't light up like it used to.

**Comment by aalm** [comment]
I've ran 30A on BBS02 750w 48v without any issues(other than temperature reaching the 85c limit after about 10km on full throttle alone).

**Comment by aaron-manning** [comment]
I have removed the motor from the bike and found that the main wires coming out of the controller have been crushed between the motor and the frame.

**Comment by aaron-manning** [comment]
I cut out the crushed cable section and re-wired it. Sadly the problem remains.

I connect the (fully charged) battery, and there's no power. programming cable does not light up, throttle does nothing.

I have tested the controller with a multimeter and everything seems fine. No shorts between each phase and positive/negative.
I am unsure what to do at this point.

**Comment by aaron-manning** [comment]
There's power through the rest of the harness.
The MOSFETs seem fine on the controller but it is otherwise dead.

**Comment by aaron-manning** [comment]
Update:
My suspicion is that the bumpy trail I was riding on at the time caused the lockring securing the motor to loosen and then the main control cable that exits the controller got crushed, which seems to have caused irreversible damage to the controller.

The programming cable also seemed to have fried. It was connected to the bike at the time when the motor shut off (bike was being run without a display). I suspect whatever broke this also broke something in the controller.

I have now fitted a new controller and everything is working fine.
I think there's enough evidence that what happened cannot be blamed on running at 30a, which is good news.

**Reply by aalm** [reply]
Yep, the controller can be fried thru the harness. I'd be willing to bet my controller got the same end as yours,
but I didn't have crushed cable(I did DIY-splitter for faster programming, and sure enough I managed to cross
wrong wires or something).
It was relatively easy to "de-pot" the controller to find the burnt surfacemounted resistors(or something).
Might be possible to repair, but without knowing the exact values of lost components not worth my time,
so I'm waiting on a replacement.

**Reply by danielnilsson9** [reply]
Yes you should be really careful when doing a custom harness since the 48V line directly from battery is one of the cables, shorting that to any of the 5V logic lines will probably destroy the controller... If I remember correctly it looked like the BBSHD board had quite a bit of input protection going on, the BBS02 board was far simpler and might have less protection.
